### PR TITLE
Documentation: Add info for `multiValueFilterOperators`

### PR DESCRIPTION
**Pull request by ashharrison90**
**What this PR does / why we need it**:

- updates the plugin metadata documentation with information about the new `multiValueFilterOperators` property

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
idk whether to mark this is `patch` or `no-changelog`, defer to you 🙇

### CODE REVIEW

**Review by sunker — COMMENTED** [review]
LGTM, but I'll let @josmperez do the final review.

File: docusaurus/docs/reference/metadata.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The `plugin.json` file is required for all plugins. When Grafana starts, it scan
...
-| `preload`                   | boolean                       | No       | Initialize plugin on startup. By default, the plugin initializes on first use, but when preload is set to true the plugin loads when the Grafana web app loads the first time. Only applicable to app plugins.                                                                                                                                                                                          |
+| `preload`                   | boolean                       | No       | Initialize plugin on startup. By default, the plugin initializes on first use, but when preload is set to true the plugin loads when the Grafana web app loads the first time. Only applicable to app plugins. When setting to true implement [frontend code splitting](../get-started/best-practices.md#app-plugins) to minimise performance implications.                                             |
```
Comment (josmperez) [inline docusaurus/docs/reference/metadata.md:41]:
```suggestion
| `preload`                   | boolean                       | No       | Initialize plugin on startup. By default, the plugin initializes on first use, but when preload is set to true the plugin loads when the Grafana web app loads the first time. Only applicable to app plugins. When setting to `true`, implement [frontend code splitting](../get-started/best-practices.md#app-plugins) to minimise performance implications.                                             |
```

**Review by josmperez — APPROVED** [review]
LGTM.

**Comment by josmperez** [comment]
The no-changelog tag is correct. Doc updates don't get to the changelog normally.

**Pull request merged**